src/components/plot-list/index.tsx

```diff
@@ -31,8 +35,8 @@ export class PlotListBase extends React.PureComponent<PlotListProps, any> {
   }
 
   public render() {
-    const {plots, handleAction, bookmark, limit} = this.props;
-    const plotListItems = plots.slice(0, limit).map((plot, index) => {
+    const {plots, handleAction, bookmark, limit, isLoading} = this.props;
+    const plotListItems = plots ? plots.slice(0, limit).map((plot, index) => {
```

Comment:
You can do `plots && ...components...` (react's preferred way)  instead of `plots ? ...components... : null`